src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -128,8 +150,29 @@ const AppContent = React.memo(({ repo, repoVersions, onRequestUpgrade }) => {
                     'No source code link.'
                   )}
                 </div>
+                {!!repoDetails && (
+                  <React.Fragment>
+                    <Heading2>Details</Heading2>
+                    <Markdown>
+                      {
+                        remark()
+                          .use(remark2react)
```

Comment:
I saw in https://github.com/remarkjs/remark-react#optionsremarkreactcomponents that we can override some of the default components. The only one I can really think of that we might want to consider is replacing `a` for `SafeLink`.

Reply:
Good call!